public/app/features/alerting/unified/components/receivers/ReceiversTable.tsx

```diff
@@ -169,6 +169,7 @@ function LastNotify({ lastNotifyDate }: { lastNotifyDate: string }) {
 }
 
 function NotifiersTable({ notifiersState }: NotifiersTableProps) {
+  const durationIsNull = (duration: string) => duration === '0s';
```

Comment:
Maybe we should use `parseInt(...)` here and check the numeric value? That would also work for `0, 0ms, 0d` etc

Reply:
> Maybe we should use `parseInt(...)` here and check the numeric value? That would also work for `0, 0ms, 0d` etc

good idea 👍